### PR TITLE
tweaked the german language file

### PR DESCRIPTION
**Pull request by manuelgu**

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Thanks!

File: resources/lang/de/dashboard.php [hunk 1]
```diff
@@ -118,7 +118,7 @@
     // Subscribers
     'subscribers' => [
         'subscribers'  => 'Abonnenten',
-        'description'  => 'Abonnenten erhalten e-Mail-Updates, wenn Vorfälle erstellt werden.',
+        'description'  => 'Abonnenten erhalten E-Mail Updates wenn Vorfälle erstellt werden.',
```
Comment (ApfelUser) [inline resources/lang/de/dashboard.php]: One question here for consistency, as I have stumbled over this several times. How do we spell "E-Mail"? My preference would be indeed **"E-Mail"**, but the translator before me used **"eMail"** in Transifex, so I continued with that. But this line now changes that. So which one do you guys prefer:

  * **E-Mail**
  * **Email**
  * **eMail**
  * **e-Mail**
  * **email**

... Maybe some possible spelling option I forgot?

**Comment by manuelgu** [comment]
According to Duden (http://www.duden.de/rechtschreibung/E_Mail), E-Mail is the correct spelling.

**Comment by ApfelUser** [comment]
@manuelgu Yeah, I just checked that now too :+1:  I guess I'll change everything to E-Mail then.

**Pull request merged**